site-structure.json

```diff
@@ -150,6 +189,7 @@
                     { "pageName": "Uploader", "href": "/UploaderAPI.html" },
                     { "pageName": "Undo", "href": "/to-do/UndoAPI.html" },
                     { "pageName": "Table of Contents", "href": "/to-do/TableOfContentsAPI.html" },
+                    { "pageName": "ChangeApplier API", "href": "/ChangeApplierAPI.html" },
```

Comment:
I would prefer to leave it for now rather than make too many changes. Definitely worth looking at later. Thoughts?